packages/flutter/lib/src/material/checkbox.dart

```diff
@@ -200,7 +200,7 @@ class _RenderCheckbox extends RenderToggleable {
         activeColor: activeColor,
         inactiveColor: inactiveColor,
         onChanged: onChanged,
-        size: const Size(2 * kRadialReactionRadius, 2 * kRadialReactionRadius),
+        size: const Size(2 * kRadialReactionRadius + 8.0, 2 * kRadialReactionRadius + 8.0),
```

Comment:
Is there value in maybe adding a comment to explain where the 8.0 is coming from here and in other places?

Reply:
The + 8.0 serves to increase the radius from 20 to 24. We should not increase the radius from 20 to 24.